src/content/guides/production.md

```diff
@@ -200,12 +200,13 @@ __webpack.prod.js__
     plugins: [
       new UglifyJSPlugin({
         sourceMap: true
-      }),
+-     })
++     }),
 +     new webpack.DefinePlugin({
 +       'process.env.NODE_ENV': JSON.stringify('production')
 +     })
     ]
-  })
+  });
```

Comment:
Hi @Simek, thanks for your contribution. As for webpack v4 we want users to rely on `mode` and not define manually the `NODE_ENV` variable since the default produciton mode does it for them.

Can you change this code snippet? After that I think it would be good to merge.

Reply:
Even if you can do some proff reading to the article and see if we can tip that feature, bonus points :)

Reply:
@Simek Any update on this? :)

Reply:
@jackfranklin @dhruvdutt Can you clarify what exact changes you would like me to implement in this PR? I've seen @chenxsan and @RasmusFonseca have added commits/small fixes there and your comments seems for me related to that portion of changes.

Reply:
My comment is that the guide suggests the user to manually configure uglify plugin and `NODE_ENV` while `mode: 'production'` does all that out of the box.

Reply:
@jackfranklin I've know that since v4 there is a simplified `mode` option, but how's it's related to clarifying "open" option for devServer? This shouldn't be fixed or resolved in a separate Issue/PR?

Reply:
Ugh sorry GitHub is double highlighting the diff on the documentation so I though you were adding those lines, my bad. Also My handle @jeremenichelli, let's not spam Jack here 😉